scripts/ci-conformance.sh

```diff
@@ -82,6 +82,7 @@ export AZURE_LOCATION_EDGEZONE="${AZURE_LOCATION_EDGEZONE:-$(capz::util::get_ran
 export AZURE_CONTROL_PLANE_MACHINE_TYPE="${AZURE_CONTROL_PLANE_MACHINE_TYPE:-"Standard_B2s"}"
 export AZURE_NODE_MACHINE_TYPE="${AZURE_NODE_MACHINE_TYPE:-"Standard_B2s"}"
 export WINDOWS="${WINDOWS:-false}"
+export AZWI_RESOURCE_GROUP=capz-wi-$(capz::util::random_suffix)
```

Comment:
we don't want `export AZWI_RESOURCE_GROUP="${AZWI_RESOURCE_GROUP}:-"capz-wi-"$(capz::util::random_suffix)"` ?